compiler/CCT/fail/H_L_L.jazz

```diff
@@ -1,8 +1,9 @@
-fn add_H_L_L(#secret reg u64 x, #public reg u64 ya) -> #public reg u64 {
+export fn add_H_L_L(#secret reg u64 x, #public reg u64 ya) -> #public reg u64 {
```

Comment:
Can you explain why you have change all the tests.
What is the motivation of adding the export.

Reply:
If they are not marked `export` they are not present at the later compiler passes where DOIT checking happens :)

Reply:
Then those programs are not suited for testing the new functionality.

Reply:
Well, AFAIK the `export` does not change anything about them with regards to the CT checking (the ordinary tests would have caught that) so I believe these can be reused for DOIT checking. They should be reused to see that unless specific non-DOIT instructions are used DOIT checking is the same as CT checking.

This brings me to a broader point that DOIT-specific tests should be added, which I will work on.

Reply:
Did you manually inspect the error messages of the CT checker before and after your change to validate that they are still the same? Do you expect the reviewers of your PR to do this boring work?

Reply:
I would expect the tests (and some understanding of what the `export` keyword does) to ensure that. Nevertheless I just manually checked and there is no change in the ordinary CT checker output.